ci-operator/step-registry/single-node/conf/aws/single-node-graceful-shutdown-conf-e2e-commands.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
This file name doesn't match single-node-graceful-shutdown-conf-aws-commands.sh.

I don't think we need the e2e/single-node-graceful-shutdown script either.

We will need the logic in this file as well: https://github.com/openshift/release/blob/51d1f44a3944759aa55bb3a5a20d44169da8d6ec/ci-operator/step-registry/single-node/conf/aws/single-node-conf-aws-commands.sh